src/constants/index.js

```diff
@@ -5,10 +5,6 @@ import {
 
```

Comment:
Since you created contacts file userFacing.js, I think this should be renamed to utilityConstants.js or something.  The name index.js is just confusing at this point, since it is not an entry point for all of our constants anymore.